tests/print_str.py

```diff
@@ -0,0 +1,13 @@
+def printCompOperations():
+   print("x < y")
+   print("x !< y")
+   print("x > y")
+   print("x !> y")
+   print("x == y")
+   print("x != y")
+
+
+def main0():
+   printCompOperations()
+
+main0()
```

Comment:
Why we cannot have it as an integration test?

Reply:
The bug is during decoding of `wasm`. The simplest case where decoding of `wasm` takes place is during `wat` generation. Hence, I added the test as `wat` reference test.

Another case where decoding of `wasm` takes place is in the `wasm_x64`/`wasm_x86` backends during `x64/x86` generation. It seems possible to add it as an integration test by adding the same test and enabling the `wasm_x64` backend. But it seems that might indicate that the issue/bug/corner-case (that the test file is testing) is in the `wasm_x64` backend during `x64` generation. In actual, the issue is in `wasm` decoding, so it seems more natural to add as a `wat` reference test.